### ISSUE TITLE
Unable to deal with passphrase-protected SSH keys

### ISSUE THREAD

**Issue by eliasp** [xetys/hetzner-kube#1]
It looks, like `hetzner-kube cluster create […]` is unable to deal with passphrase-protected SSH keys.

After successfully initializing all servers, the last 2 lines of output were:
```
2018/01/26 09:11:28 installing docker.io and kubeadm on node 'grfy-k8s-dc5d-master-01'...
2018/01/26 09:11:28 parse key failed:ssh: cannot decode encrypted private keys
```

Using `hetzner-kube` eb0790b

**Comment by eliasp** [comment]
Just confirmed this by creating a new unprotected keypair and used this for a test-cluster, where everything worked as expected…

**Comment by xetys** [comment]
thanks for the report!

I'll need to check out how to make golang.org/crypto/ssh work with passphrase connections...or if you know how to make this, you maybe could do a PR?

**Comment by cice** [comment]
i was bored, so i implemented it here #8

**Comment by xetys** [comment]
closed by #8
